### PR TITLE
Add aarch64

### PR DESCRIPTION
**Pull request by leofang**
Follow-up of https://github.com/conda-forge/cudatoolkit-feedstock/pull/59 and https://github.com/conda-forge/docker-images/pull/189.

Also add myself as a maintainer in this PR 🙂

Note: conda-smithy seems to have a bug in generating README contents for compilers. For example, the README for this feedstock is already incorrect --- neither ppc64le nor win64 is listed. With this PR it gets worse: osx-arm64 is listed (which we don't build for). 

I don't think it hurts, though. We can fix conda-smithy separately and then rerender this feedstock.

cc: @jakirkham 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge/core The `linux-anvil-aarch64-cuda` docker images seem to have permission issues. They were uploaded after https://github.com/conda-forge/docker-images/pull/189 was merged, but they are not accessible by the CI nor human users.

**Comment by leofang** [comment]
截圖 2021-09-27 00 16 59

**Review by kkraus14 — APPROVED** [review]
LGTM, thanks @leofang

**Comment by leofang** [comment]
@conda-forge-admin, please restart ci

**Comment by leofang** [comment]
> The `linux-anvil-aarch64-cuda` docker images seem to have permission issues. They were uploaded after [conda-forge/docker-images#189](https://github.com/conda-forge/docker-images/pull/189) was merged, but they are not accessible by either the CI or human users.

@kkraus14, did you or someone fix the permission issue? Looks like it's working now --- all tests passed!

**Comment by isuruf** [comment]
Yes, I have to fix it manually every time there's a new image name.

**Comment by leofang** [comment]
Ah, many thanks @isuruf! 👍

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,9 @@ package:
 
 build:
   number: {{ number }}
-{% if cuda_major == 0 or (target_platform == "linux-ppc64le" and cuda_major_minor != (10, 2)) %}
+{% if cuda_major == 0
+    or (target_platform == "linux-ppc64le" and cuda_major_minor != (10, 2))
+    or (target_platform == "linux-aarch64" and (cuda_major != 11 or arm_variant_type != "sbsa")) %}
```
Comment (jakirkham) [inline recipe/meta.yaml:21]: Could we please write this in terms of `(10, 2)` as well? Want to make sure this still works smoothly as future versions come out :)

**Review by jakirkham — COMMENTED** [review]
Thanks Leo! 😄

Had one suggestion below

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (leofang) [reply recipe/meta.yaml]: I wasn't thinking before changing... @jakirkham What do you expect? `cuda_major_minor < (11, 0)`?

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Yeah that seems reasonable. If we make that change, can we please do the same thing for `ppc64le`?

Comment (leofang) [reply recipe/meta.yaml]: Sure.

Comment (leofang) [reply recipe/meta.yaml]: @jakirkham I guess I misunderstood what you meant. For ppc64le let's keep it as is until we add CUDA 11 support for it, is that ok?

**Review by Ethyling — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Yeah no worries. That's reasonable. This is admittedly a bit unrelated. Just was thinking on how we might tidy things while we were here

**Review by jakirkham — APPROVED** [review]
Thanks Leo! 😄

**Comment by leofang** [comment]
Thanks @kkraus14 @isuruf @Ethyling @jakirkham!

**Pull request merged**